airflow-core/src/airflow/api_fastapi/core_api/services/ui/connections.py

```diff
@@ -68,6 +68,7 @@ def __init__(
             description: str = "",
             default: str | None = None,
             widget=None,
+            source: Literal["Dag", "task"] | None = None,
```

Comment:
Can we change either to `dag` or `Task`? The mismatched case is very annoying.

(I quickly searched the code base, literal strings tend to be either all-lowercase or ALL-UPPERCASE.)